### DISCUSSION TITLE
Feature Request: Can we add features with the ability to chat with AI with an option to view embedding results

### DISCUSSION THREAD

**Discussion by sungkim11** [danny-avila/LibreChat#154] [Feature Requests & Suggestions]
This feature request would be helpful if we can extend the functionality to chat with embedding where:
- User clicks on LLM response
- It expands on the right nav to detail out which embedding results the LLM response was derived from
- Embedding responses include both metadatas as well as text. There can be multiple embedding responses.

**Comment by danny-avila** [comment]
I’m not sure I follow. Currently there is no use of vector embeddings, or did you mean something else? I think before this, use of vector embeddings will have to come first.

if you meant embedded information, like metadata as you mentioned, could you give more clarity about that

**Comment by sungkim11** [comment]
Let's say I were to use app using openai api where it prompts the model to search vector embeddings and find an answer. I would like an option to view those embedding results. Those embedding results will be in a form of pandas dataframe for example.

**Comment by danny-avila** [comment]
That's a great idea, definitely doable. I do plan on integrating embeddings for chat use cases such as you described. Moving this to discussion as I'm keeping note of feature requests there.
